modules/sql/src/main/resources/db/migration/V059__Niri_KeywordNames.sql

```diff
@@ -0,0 +1,47 @@
+---------------------------
+-- NIRI FITS Keyword Names
+---------------------------
+
+INSERT INTO e_fits_keyword_names(id, name)
+VALUES
+	('COADDS', 'COADDS'),
```

Comment:
The all caps is kind of Java-ish and we've avoided it elsewhere but I think it may be expected if they are FITS keywords.

Reply:
Yes, that is exactly the case.